packages-exp/app-exp/src/errors.ts

```diff
@@ -31,7 +31,8 @@ const ERRORS: ErrorMap<AppError> = {
     "No Firebase App '{$appName}' has been created - " +
     'call Firebase App.initializeApp()',
   [AppError.BAD_APP_NAME]: "Illegal App name: '{$appName}",
-  [AppError.DUPLICATE_APP]: "Firebase App named '{$appName}' already exists",
+  [AppError.DUPLICATE_APP]:
+    "Firebase App named '{$appName}' already exists with different options",
```

Comment:
"different options or config"? or are we just using "options" as a general term to mean like params?